plugin/Gui/AdminPageApiSettings.php

```diff
@@ -32,6 +32,7 @@
 use function get_option;
 use function json_encode;
 use onOffice\WPlugin\Utility\SymmetricEncryption;
+use PHP_CodeSniffer\Exceptions\RuntimeException;
```

Comment:
Please use PHP's `\RuntimeException` here

Reply:
Or remove the `use` import, since it's not in use anyway

Reply:
I removed it